README.md

```diff
@@ -78,41 +84,44 @@ https://github.com/flashbots/mev-boost/releases
 
 Requires [Go 1.18+](https://go.dev/doc/install).
 
-### `go install`
+**`go install`**
 
-Install mev-boost with `go install`:
+Install MEV-Boost with `go install`:
 
 ```bash
 go install github.com/flashbots/mev-boost@latest
 mev-boost -help
 ```
 
-### Clone & build
+**Clone and Build
+**
+Ensure you are downloading the most updated MEV-Boost release. Releases are available at https://github.com/flashbots/mev-boost/releases
 
 clone the repository and build it:
 
 ```bash
 git clone https://github.com/flashbots/mev-boost.git
 cd mev-boost
+# Build the checked-out version from https://github.com/flashbots/mev-boost/releases
```

Comment:
that comment is not quite correct, please see my suggestion in the PR before and use that here